### PR TITLE
Clean up timeline

### PR DESCRIPTION
**Pull request by Joltrix**
This provides a fix for #34 

The Sort Buttons should now also get properly sorted when a Filter gets deleted (If for example EventFilter X is selected, and then the last event of that type is deleted, the "ALL" option should be selected automatically).

### CODE REVIEW

**Comment by Joltrix** [comment]
@bouguila Can you check to see if the Timeline works as intended? specifically regarding the filtering.

File: src/Timeline/wwwroot/Timeline/views/ActionsPage.html [hunk 1]
```diff
@@ -119,6 +119,7 @@
             };
 
             template.checkIfSelected = function (state) {
+                debugger;
```
Comment (bouguila) [inline src/Timeline/wwwroot/Timeline/views/ActionsPage.html]: please remove the debugger

Comment (Joltrix) [reply src/Timeline/wwwroot/Timeline/views/ActionsPage.html]: Oh! I had totally missed that one! Thank you! :)

**Comment by bouguila** [comment]
LGTM

**Pull request merged**